aleph-client/Cargo.toml

```diff
@@ -10,11 +10,10 @@ anyhow = "1.0"
 codec = { package = 'parity-scale-codec', version = "3.0.0", features = ['derive'] }
 hex = { version = "0.4.3", features = ["alloc"] }
 log = "0.4"
+thiserror = "1.0.39"
```

Comment:
this change makes little sense - specifying 1.0 will already pull that version if it's the newest

Reply:
You're right. Is it okay if I leave it though or would you prefer for me to revert it?

Reply:
revert please :raised_hands:

Reply:
Reverted.